requests_toolbelt/adapters/ssl.py

```diff
@@ -48,3 +54,7 @@ def init_poolmanager(self, connections, maxsize, block=False):
             maxsize=maxsize,
             block=block,
             ssl_version=self.ssl_version)
+
+    def proxy_manager_for(self, *args, **kwargs):
+        kwargs['ssl_version'] = self.ssl_version
```

Comment:
Oh wait no that's not true. On some versions it does. Good idea, I'll make the definition of this method conditional.